code/modules/events/market_crash.dm

```diff
@@ -23,7 +23,7 @@
 	var/list/poss_reasons = list("the alignment of the moon and the sun",\
 		"some risky housing market outcomes",\
 		"the B.E.P.I.S. team's untimely downfall",\
-		"speculative SolFed grants backfiring",  /*SKYRAT EDIT CHANGE; original was "speculative Terragov grants backfiring"*/\
+		"speculative TerraGov grants backfiring",  /*SKYRAT EDIT CHANGE; original was "speculative Terragov grants backfiring"*/\
```

Comment:
You can remove the Skyrat Edit marker if this is just restoring it to its original state.